modules/nf-core/abacas/meta.yml

```
@@ -46,11 +50,15 @@ output:
             'test.abacas.tab', 'test.abacas.unused.contigs.out',
             'test.abacas.MULTIFASTA.fa' ]
           pattern: "*.{abacas}*"
+          ontologies: 
+            - edam: "http://edamontology.org/format_1929" # FASTA
   - versions:
       - versions.yml:
           type: file
           description: File containing software versions
           pattern: "versions.yml"
+          ontologies: 
+            - edam: "http://edamontology.org/format_3750" # Ontology data
```

This comment applies to all other yaml files :)
```suggestion
            - edam: "http://edamontology.org/format_3750" # YAML
```